wpeverest/everest-forms
# Feature - Salesflare into Module

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .

### How to test the changes in this Pull Request:

1. Go to **EVF Dashboard** >  **Features**
2. Enable the **Salesflare** feature
3. Go to the global settings of salesflare from the small settings button
4. Enter the **API key and its nickname**.
5. Now go to **Everest Forms Settings> Add New Form**
6. Go to **Integrations** from the form Settings and map the corresponding fields.
7. And click on **Save** button.
8. Finally, fill-up the form and check whether the data are mapped accordingly or not.

PR Dependency [Salesfare](https://github.com/wpeverest/everest-forms-pro/pull/899)
[Docs for API key generation](https://api.salesflare.com/docs#section/Introduction/Authentication)

### Types of changes:




### Other information:




### Changelog entry

>Feature - Salesflare into Module
